db/seeds.rb

```diff
@@ -24,8 +24,9 @@
 User.create(email: 'redactor@example.com', password: '123456', password_confirmation: '123456',
             name: Faker::Name.first_name, surname: Faker::Name.last_name,
             username: Faker::Internet.unique.username(specifier: 8), birthday: Faker::Date.birthday(min_age: 18, max_age: 65),
-            role_id: redactor_role.id)
+            role_id: redactor_role.id, confirmed_at: Time.zone.now)
 User.create(email: 'reviewer@example.com', password: '123456', password_confirmation: '123456',
             name: Faker::Name.first_name, surname: Faker::Name.last_name,
             username: Faker::Internet.unique.username(specifier: 8), birthday: Faker::Date.birthday(min_age: 18, max_age: 65),
-            role_id: reviewer_role.id)
+            role_id: reviewer_role.id, confirmed_at: Time.zone.now)
+FactoryBot.create_list(:rating, 20)
```

Comment:
Is the rating created tied to the movie and the user?

Reply:
i'll fix this

Reply:
yes it is
![image](https://user-images.githubusercontent.com/70888057/155810114-8a888e71-01d3-49ef-9fe7-17ef34edccdd.png)